### ISSUE TITLE
Notifications: Hyperlink to the user who the comment relates to

### ISSUE THREAD

**Issue by mikeallbutt** [DiscipleTools/disciple-tools-theme#875]
Like in the attached image, have the @Someone be a link to that user or contact. This will make it easier to move around the system, as you might need to take action related to what the notification relates to.

![Screenshot 2019-12-10 21 03 23](https://user-images.githubusercontent.com/43966676/70588092-4cbec500-1b91-11ea-8e28-c0370f370611.png)

**Comment by corsacca** [comment]
Can you give an example of a notification where the user would want to see the contact of the @mentioned user instead of the contact or group in question?

**Comment by mikeallbutt** [comment]
> Can you give an example of a notification where the user would want to see the contact of the @mentioned user instead of the contact or group in question?

Yes I can. I have a leader who uses DT to write notes about the groups they are leading. I want to give feedback to this leader but not via the group page that is referenced. By having the @user as a link of person who wrote the comment, will allow me to go to their contact page and leave a comment. 

I could also write a comment on my profile and @metion them, but getting to my profile is also not an easy route since there is no link to this in my user profile page.
